changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.87.2
  * Fixed a bug when RealmObjectSchema.addField called with PRIMARY_KEY modifier, the field was not set as a required field (#2001).
+ * Fixed a bug which could cause ConcurrentModificationException when get a new RealmObject in RealmObject's change listener (#1970).
```

Comment:
@cm what about:
Fixed a bug which could cause ConcurrentModificationException when manipulate with RealmObject in RealmObject's change listener (#1970).